tags/resources/tag-formation-templates/template-roles.md

```diff
@@ -1,3 +1,103 @@
+* Serving as TAG leadership representative in CNCF project discussions and on community events.
+* Onboard and mentor new community members.
+* Mentor community members in a leadership role within the TAG.
+* Enforce and encourage diversity in the TAG efforts.
+* Enforce and encourage good communication in the TAG efforts following the [CNCF CoC](https://www.cncf.io/conduct/).
+* Support the TAG Chairs.
+
+## Working Group Lead
+
+A working group lead is an experienced contributor to the TAG that applies to lead along one or two others a working group.
+Working groups as community governance structure are defined in the [CNCF TAG](https://github.com/cncf/toc/blob/master/tags) folder.
+TAG Chairs are responsible facilitating the working group applications and make sure that the process is done transparent, accessible and traceable for everyone.
+One of the TAG chairs serves as the working group sponsor and provides guidance to the working group leads.
+Working group leads are signed off by the TAG Chairs and the TOC Liaisons.
+The [leadership election process is described here](template-leadership-election-process.md#working-group-lead).
+
+**A working group lead is expected to**:
+* Schedule, host, plan and facilitate meetings for the working group.
+* Provide technical direction unique to the working group.
+* Plan working group deliverables.
+* Report to the TAG on the working group status.
+* Evolve the working group.
+* Garner active participation.
+* Establish documents and correct permissions for contributions to occur.
+* Contribute content to the working group.
+* Onboard and setup the working group lead successor.
+
+## Project Lead
+
+A project lead manages a community group for a specified period of time and works to achieve a specific outcome that has been previously discussed and agreed upon.
+Unlike working groups, projects require a predetermined time frame and are limited in scope and focused on a central deliverable. The project proposal must be discussed at a meeting and actively communicated to the entire TAG community.
+A TAG Chair or TL acts as a sponsor of the project and dedicates a portion of their time to actively support the effort.
+TAG Chairs must sign off on the establishment of the project group.
+Due to the limited scope of the role, there are no specific requirements for structuring the selection process for the project lead.  
+The TAG Chairs are responsible for selecting a suitable project lead and must ensure that the establishment of the project is transparent, accessible, and understood by all.
+
+**A project lead is expected to**:
+* Schedule, host, plan and facilitate meetings for the project.
+* Provide technical direction unique to the project.
+* Plan the project deliverable.
+* Report to the TAG on the project status.
+* Garner active participation.
+* Establish documents and correct permissions for contributions to occur.
+* Contribute content to the project.
```

Comment:
Let's consolidate this with the Working Group Leads under a Group Lead function. One may be time-limited/deliverable oriented, the other may be focused groups for ongoing and continual activities. So long as these are defined by the TAG, and approved by the TOC liaisons, the general responsibilities are not much different except in the specifics of implementation.

Reply:
Yes, good idea. I will merge both sections. 👍